lib/prosperworks/User.rb

```diff
@@ -0,0 +1,12 @@
+module ProsperWorks
+  class User < BaseEntity
```

Comment:
`BaseEntity` has more fields than the `User` model needs, so this should inherit instead from `Base`